File: zeyigou_cart_web/src/main/webapp/js/service/cartService.js
app.service("cartService",function ($http){
    //1.查看购物车列表
    this.findCartList=()=>{
        return $http.get("./cart/findCartList.do");
    }

    //2.添加到购物车
    this.addCartList=(itemId,num)=>{
        return $http.get("./cart/addCartList.do?itemId"+itemId+"&num="+num);
    }
})

File: zeyigou_cart_interface/src/main/java/com/zeyigou/cart/service/CartService.java
package com.zeyigou.cart.service;

import com.zeyigou.group.Cart;

import java.util.List;

public interface CartService {
    List<Cart> getCartListFromRedis(String name);

    List<Cart> mergeCartList(List<Cart> cookieCartList, List<Cart> redisCartList);

    void saveCartListToRedis(List<Cart> redisCartList, String name);

    List<Cart> addCartList(List<Cart> cartList,Long itemId, int num);
}
